src/PageRouter.js

```diff
@@ -0,0 +1,36 @@
+import { React } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import HomePage from "./components/pages/HomePage";
+import WelcomePage from "./components/pages/WelcomePage";
+import CreateInvestorPage from "./components/pages/CreateInvestorPage";
+import BacktestingPage from "./components/pages/BacktestingPage";
+import TransactionHistoryPage from "./components/pages/TransactionHistoryPage";
+import AccountPage from "./components/pages/AccountPage";
+import SignInPage from "./components/authentication/SignInPage";
+
+export function PageRouter() {
+  return (
+    <BrowserRouter>
```

Comment:
I think there are a lot of missing routes here. Based on Figma I think there needs to be routes for

/investor - view an investor
/job - view a job
/backtests - view results/list of all backtests
/createbacktest - create a new backtest
/backtestresult - view the results of a specific backtest
/changepassword - change a users password

Reply:
Since these are nested pages I didn't worry about these until it comes time to actually create them. I assumed this story was for the basic initial pages the user can access through navbar and sidebar.

Reply:
Okay those can be added down the road if you would prefer - Will went ahead and added all pages and nested pages as part of his story so I didn't know if you wanted to do the same.